requirements/base.txt

```diff
@@ -1,13 +1,11 @@
-# SHA1:728467968860943ba7eecd70ca33a9034bbc00ad
 #
-# This file is autogenerated by pip-compile-multi
```

Comment:
You didn't use `pip-compile-multi`?

Reply:
Didn't, just about to fix that, forgot we use it now.